<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>完全消去プログラム</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeWidget" name="storagePanel">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>40</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="headerShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ローカルディスク</string>
     </property>
    </column>
    <item>
     <property name="text">
      <string>Physical HDD</string>
     </property>
     <item>
      <property name="text">
       <string>Local Disk (C:)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Local Disk (D:)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Local Disk (E:)</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>Removal Storages</string>
     </property>
     <item>
      <property name="text">
       <string>USB Port 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>USB Port 2</string>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>アップロード</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>571</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="sysInfo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>FTP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SCP</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>351</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>消去方法</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startWipe">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>削除開始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>終了</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>最新の情報に使新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ドライブの</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ハードウェア情報</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>パーティション情報</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="driveInfo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>351</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <attribute name="headerDefaultSectionSize">
     <number>83</number>
    </attribute>
    <attribute name="headerStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>ドライブの</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>フォーマット</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ボリコームラべル</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>サイズ</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="wipeMethod">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>AllZero(0)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>指定の数値</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>指定回数繰り返し</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="wipeOption">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>DoD 5220.22-M 米国国防総省方式（上書き3回）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>米国家安全保障局(NSA)推奨方式（上書き3回）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AR380-19 米陸軍方式（上書き3回）</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionAbout_this">
   <property name="text">
    <string>About this</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
